scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixFileEvaluation.java

```diff
@@ -18,11 +19,22 @@
 
     /**
      *
-     * @return boolean true if there is no error when scalafix has been evaluated on the file
+     * @return boolean true if there is no error except LintError when scalafix has been evaluated on the file
      */
     boolean isSuccessful();
 
-    ScalafixError[] getErrors();
+    /**
+     *
+     * @return a more detailed error in case of scalafix.
+     */
+    Optional<String> getErrorMessage();
```

Comment:
in case of what? I think this adds confusion rather than thelp the user
```suggestion
    Optional<String> getErrorMessage();
```